### PR TITLE
Add step with link to Yellow installer

### LINKED ISSUE #119: Please provide download link for installer
Please provide a link to, for example, https://github.com/NabuCasa/yellow-buildroot/releases so those who don't have a supported OS for rpi-imager may find the files and manually flash their install media.

Background, I use Fedora at home. Outside of forcing the package with alien, I can't install rpi-imager because it only provides packages for Ubuntu, Windows, and Mac. The only install instructions on the yellow.home-assistant.io site point to finding the images in rpi-imager.

### PR DESCRIPTION
**Pull request by c0ffeeca7**
- required for users who do not use Raspberri Pi Imager.
- closes #119

### CODE REVIEW

File: src/procedures/procedures.11tydata.yaml [hunk 1]
```diff
@@ -149,6 +149,14 @@ procedureInstallNvme:
...
   title: Installing Home Assistant Software on Kit
   steps:
+    - title: Installing the Home Assistant OS Installer
+      image: balena.png
+      content: |
+        <ul>
+        <li>This procedure shows how to install the Home Assistant OS Installer for Yellow using Raspberri&nbsp;Pi&nbsp;Imager. If you are using another imager, download the latest Home&nbsp;Assistant&nbsp;Yellow Installer here:
+         <a href="/power-supply/#eject-the-usb-flash-drive" target="_blank">https://github.com/NabuCasa/yellow-buildroot/releases </a></li>
+          <li>Write the image to your USB flash drive and continue with the step <a href="/power-supply/#eject-the-usb-flash-drive" target="_blank">eject the usb flash drive</a>.</li>
+        </ul>
```
Comment (agners) [inline src/procedures/procedures.11tydata.yaml:159]: Making this a separate step will confuse a lot of people not reading the instructions exactly. I'd suggest to add an info point in the "Installing the Raspberry Pi Imager" step, saying something like:


* If your platform doesn't support the Raspberry Pi Imager you can also use another imager (e.g. balenaEdger) and flash the Home Assistant OS Installer for Yellow directly from [GitHub releases](https://github.com/NabuCasa/yellow-buildroot/releases).

Comment (c0ffeeca7) [reply src/procedures/procedures.11tydata.yaml]: done. I had something like that at first, then moved it out into a separate step :-).

Comment (agners) [reply src/procedures/procedures.11tydata.yaml]: Can you also drop `public/images/balena.png`, it is no longer required.

File: src/procedures/procedures.11tydata.yaml [hunk 2]
```diff
@@ -154,6 +154,7 @@ proceduresInstallingHAOS:
       content: |
         <ul>
           <li>Install the Raspberry Pi Imager on your computer as described under <a href="https://www.raspberrypi.com/software/" target="_blank">https://www.raspberrypi.com/software/</a></li>
+          <li class="info">If your platform doesn't support the Raspberry Pi Imager, you can use another imager (e.g. balenaEtcher) and flash the Home Assistant OS Installer for Yellow directly from GitHub releases: <a href="/power-supply/#eject-the-usb-flash-drive" target="_blank">https://github.com/NabuCasa/yellow-buildroot/releases </a>.</li>
```
Comment (agners) [inline src/procedures/procedures.11tydata.yaml:157]:
```suggestion
          <li class="info">If your platform doesn't support the Raspberry Pi Imager, you can use another imager (e.g. balenaEtcher) and flash the Home Assistant OS Installer for Yellow directly from <a href="https://github.com/NabuCasa/yellow-buildroot/releases">GitHub releases</a>.</li>
```

**Review by agners — APPROVED** [review]

**Pull request merged**